code/widget-showcase/src/examples/list.rs

```diff
@@ -0,0 +1,26 @@
+use ratatui::{
+    prelude::*,
+    widgets::{HighlightSpacing, List, ListState},
+};
+
+pub fn render(frame: &mut Frame) {
+    let list = List::new([
+        Line::from(vec!["✔️ ".green(), "Fork tui-rs 💻".into()]),
+        Line::from(vec![
+            "✔️ ".green(),
+            "Create a ".into(),
+            "great".italic(),
+            " mascot 🐀".into(),
+        ]),
+        Line::from(vec!["✔️ ".green(), "Create a website & book 🕮".into()]),
+        Line::from(vec!["✔️ ".green(), "Celebrate 500th commit ⭐".into()]),
+        Line::from(vec!["✔️ ".green(), "Celebrate 1000th commit ✨".into()]),
+        Line::from(vec!["⌛".yellow(), "Release Ratatui 1.0.0 🎉".bold()]),
```

Comment:
Ha, pretty much this will never happen. All our deps would have to go 1.0 first.

Reply:
Yep, that was just to have something that will pretty much never change ^^